sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/SqliteCompiler.kt

```diff
@@ -85,25 +85,35 @@ class SqliteCompiler {
             .addType(table.creatorInterface())
             .addType(MapperSpec.builder(table).build())
 
+        if (table.javadoc != null) typeSpec.addJavadoc(table.javadoc)
+
         for (column in table.column_def()) {
           if (column.constantName(table.nameAllocator) == TABLE_NAME
               || column.constantName(table.nameAllocator) == CREATE_TABLE) {
             throw SqlitePluginException(column.column_name(),
                 "Column name '${column.sqliteName}' forbidden")
           }
 
-          typeSpec.addField(FieldSpec.builder(String::class.java, column.constantName(table.nameAllocator))
-              .addModifiers(PUBLIC, STATIC, FINAL)
-              .initializer("\$S", column.sqliteName)
-              .build())
+
+          val columnConstantBuilder = FieldSpec.builder(String::class.java, column.constantName(table.nameAllocator))
+                  .addModifiers(PUBLIC, STATIC, FINAL)
+                  .initializer("\$S", column.sqliteName);
```

Comment:
also kotlin doesn't need a semicolon at the end of a line. It doesn't hurt to have it but to stay consistent we've been leaving it out